src/js/media-playback-rate-button.js

```diff
@@ -20,32 +20,35 @@ class MediaPlaybackRateButton extends MediaChromeButton {
     return [
       ...super.observedAttributes,
       MediaUIAttributes.MEDIA_PLAYBACK_RATE,
-      'rates',
+      Attributes.RATES,
     ];
   }
 
+  /** @type number[] | undefined */
+  #_rates;
```

Comment:
can we leave the underscore, use `#rates` instead? 

don't really see a reason for an added underscore.

Reply:
Yup we can def drop the underscore. ❤️